instrumentation/crash/src/test/java/io/opentelemetry/android/instrumentation/crash/CrashReporterTest.java

```diff
@@ -11,49 +11,59 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import io.opentelemetry.android.instrumentation.common.InstrumentedApplication;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import io.opentelemetry.android.OpenTelemetryRum;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.logs.SdkLoggerProvider;
 import io.opentelemetry.sdk.logs.data.LogRecordData;
+import io.opentelemetry.sdk.logs.export.SimpleLogRecordProcessor;
 import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions;
-import io.opentelemetry.sdk.testing.junit5.OpenTelemetryExtension;
+import io.opentelemetry.sdk.testing.exporter.InMemoryLogRecordExporter;
 import io.opentelemetry.semconv.ExceptionAttributes;
 import io.opentelemetry.semconv.incubating.ThreadIncubatingAttributes;
 import java.util.List;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RuntimeEnvironment;
 
-class CrashReporterTest {
+@RunWith(AndroidJUnit4.class)
+public class CrashReporterTest {
+    private OpenTelemetrySdk openTelemetrySdk;
+    private InMemoryLogRecordExporter logRecordExporter;
+    private Thread.UncaughtExceptionHandler existingHandler;
 
-    @RegisterExtension
-    static final OpenTelemetryExtension testing = OpenTelemetryExtension.create();
-
-    static Thread.UncaughtExceptionHandler existingHandler;
-
-    @BeforeAll
-    static void setUp() {
+    @Before
```

Comment:
Shouldn't we be using `@BeforeEach` and `@AfterEach`?

Reply:
I'd wish, but the robolectric runner doesn't support junit5 :(